scanner/image/scanner/konflux.Dockerfile

```diff
@@ -0,0 +1,71 @@
+ARG MAPPINGS_REGISTRY=registry.access.redhat.com
+ARG MAPPINGS_BASE_IMAGE=ubi8
+ARG MAPPINGS_BASE_TAG=latest
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8-minimal
+ARG BASE_TAG=latest
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+ENV CGO_ENABLED=1
+ENV GOFLAGS=""
+ENV GOTAGS="release"
+ENV RELEASE_TAG="$CI_VERSION"
+ENV CI=1
+
+COPY . /src
+WORKDIR /src
+
+RUN scripts/konflux/fail-build-if-git-is-dirty.sh
+
+RUN make -C scanner NODEPS=1 CGO_ENABLED=${CGO_ENABLED} image/scanner/bin/scanner copy-scripts
+
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+LABEL \
+    com.redhat.component="rhacs-scanner-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="This image supports image scanning for RHACS" \
+    io.k8s.description="This image supports image scanning for RHACS" \
+    io.k8s.display-name="scanner" \
+    io.openshift.tags="rhacs,scanner,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-rhel8" \
+    source-location="https://github.com/stackrox/scanner" \
+    summary="The image scanner for RHACS" \
+    url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
+    # We must set version label to prevent inheriting value set in the base stage.
+    # TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+    version="0.0.1-todo"
+
+COPY --from=builder \
+    /src/scanner/image/scanner/scripts/entrypoint.sh \
+    /src/scanner/image/scanner/scripts/import-additional-cas \
+    /src/scanner/image/scanner/scripts/restore-all-dir-contents \
+    /src/scanner/image/scanner/scripts/save-dir-contents \
+    /src/scanner/image/scanner/bin/scanner \
+    /usr/local/bin/
+
+# The mapping files are not optional.
+# The helm chart hard codes in the indexer config the path to the mapping
+# files, which is preferred over downloading.  If the file does not exist, the
+# indexer raises an error (i.e. it does not fall back to download).
+
+COPY ./repository-to-cpe.json ./container-name-repos-map.json /run/mappings/
```

Comment:
This chunk does not exist in the downstream version of the file <https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-scanner-v4/Dockerfile.in?ref_type=heads>. How do you think, did colleagues forget to add it there?

Reply:
I should probably double check the code now, given that there has been lots of changes in the last couple of weeks.  And then I'll try to understand why it's not downstream.  It's possible it was forgotten, but I'm not sure.

Reply:
Addressed in downstream PR.